src/support/modify.c

```diff
@@ -467,15 +468,15 @@ __wt_modify_reconstruct_from_upd_list(WT_SESSION_IMPL *session, WT_CURSOR_BTREE
 
     /*
      * It is possible that a read-uncommitted reader can not reconstruct a full value. This is
-     * because another isolation snapshot writer can abort the updates in parallel and leave the
-     * reader in an invalid state by skipping a needed modify or we are not able to find a base
-     * update. It is difficult to distinguish if an aborted modify or update happened prior to the
-     * call of the function or if it has been done in parallel. Therefore there is no guarantee that
-     * a read-uncommitted reader can properly construct a full value. In this case, we will return
-     * back to the user with a retry error.
+     * because a snapshot isolation writer can abort the updates in parallel and leave the reader
+     * with an update list that does not contain enough information to reconstruct the full value.
+     * It is impossible to distinguish if an aborted modify or update happened prior to the call of
+     * the function, or if it is being done in parallel. In this case, we will return back to the
+     * user with a retry error.
```

Comment:
Also needs an update, since it's based on the other comment.

Reply:
[cfd6173](https://github.com/wiredtiger/wiredtiger/pull/10311/commits/cfd6173e791bd0fb94c3b4c7af5a952e48394496)